src/components/Room/Room.js

```diff
@@ -55,11 +53,14 @@ function Room() {
 
   // upon entering the room as a non-host, join closure stream
   useEffect(() => {
+    // re-establish room closure stream upon refresh if non-host and in the room
     if (userInRoom && !store.getState().room.userIsHost && store.getState().room.token !== null) {
       joinRoomClosureStream();
     }
+    // re-join media room upon refresh if in the room
     if (userInRoom && store.getState().room.token !== null) {
-      joinMediaRoom();
+      // TODO: test media room functionality.
+      // joinMediaRoom();
     }
   },[userInRoom]);
```

Comment:
We might need to look more into when `joinRoomClosureStream` is called. If the user is in the vestibule, they need to know that the Room is closed.
Or maybe we need to do some sort of backend call to "check if room is closed" b/c RFE still allows the user to visually enter the Room even if it got closed by the host. This is a minor edge case so I'm unsure how much it is worth investigating.

Reply:
Good catch. I will add a TODO to check this edge-case.